### ISSUE TITLE
Add social media image to page "head"

### ISSUE THREAD

**Issue by choldgraf** [jupyterhub/binderhub#903]
I've noticed that Google Collaboratory links include the Colab logo when they're shared on social media. e.g.:

https://twitter.com/GalOshri/status/1105962349310046208

I looked into it, and it seems like there are a few "meta" tags that we can add to our HTML that will cause twitter/facebook/etc to render images rather than "just" the links:

https://css-tricks.com/essential-meta-tags-social-media/

I think if we added those meta tags to the template here:

https://github.com/jupyterhub/binderhub/blob/master/binderhub/templates/page.html#L8

then we should get the same effect but for the Binder logo.

What do people think about:

1. Creating an image w/ the Binder logo and "Binder" underneath it that's fit for web viewing.
2. Added "meta" tags so that this image was displayed when people shared binder links on twitter?

I think it could be a good way to make links more discoverable and build up the brand recognition for the logo

**Comment by betatim** [comment]
I am for this! I'd start with the Binder logo by itself so that we can work on the meta tags and the logo in parallel (and don't have to wait for one being done before we can tackle the other).

**Comment by choldgraf** [comment]
hmmm, @betatim maybe this is why it didn't work? https://stackoverflow.com/questions/9858577/open-graph-can-resolve-relative-url

**Comment by choldgraf** [comment]
just merged in another PR to try fixing this with an absolute URL. Also added in some extra meta tags because [this facebook post](https://developers.facebook.com/docs/sharing/best-practices/#precaching) said it was a good idea to get the images to show up

**Comment by choldgraf** [comment]
it works now! https://twitter.com/choldgraf/status/1155503042621366272
